### PR TITLE
Python examples

### PR DESCRIPTION
**Pull request by owentmfoo**

### CODE REVIEW

File: Python/pylintrc
Comment (salman-mclarenapplied) [inline Python/pylintrc:1]: Is this file needed for the sample code?

File: Python/stream_reader_sqlrace/additioanl_requirements.txt
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/additioanl_requirements.txt:1]: Should the name of the file be additional_requirements.txt?

File: Python/stream_reader_sqlrace/main.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        packet_type = new_packet.type
+        content = new_packet.content
+        session_key = new_packet.session_key
+
+        # discard the packet if the session key does not match
+        if match_session_key and (session_key != self.session_key):
+            logger.info("Session key mismatch, packet discarded.")
+            return
+
+        # try and deserializes the packet, if the pack type can be located.
```
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/main.py]: should say `packet` instead of `pack`

File: Python/stream_reader_sqlrace/main.py [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        # if we can't find a corresponding packet type, discard the packet.
+        try:
+            packet_class = getattr(open_data_pb2, packet_type + "Packet")
+            packet = packet_class.FromString(content)
+        except AttributeError:
+            logger.debug(
+                "Unable to deserializes pack content for pack type %s", packet_type
```
Comment (salman-mclarenapplied) [inline Python/stream_reader_sqlrace/main.py]: same as above

File: Python/stream_reader_sqlrace/atlas_session_writer.py [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+from typing import List
+
+from ma.streaming.open_data.v1 import open_data_pb2
+from stream_reader_sqlrace.sql_race import SQLiteConnection
+
+logger = logging.getLogger(__name__)
+# configure pythonnet runtime for SQLRace API
+os.environ["PYTHONNET_RUNTIME"] = "coreclr"
+os.environ["PYTHONNET_CORECLR_RUNTIME_CONFIG"] = (
+    r"C:\Program Files\McLaren Applied Technologies\ATLAS 10\MAT.Atlas.Host.runtimeconfig.json"
```
Comment (TimOliverMAT) [inline Python/stream_reader_sqlrace/atlas_session_writer.py]: Why do we reference the A10 runtime config? Also, as you have an A10_INSTALL_PATH variable below, may be better to use that for the path to ensure they never get out of sync?

Comment (owentmfoo) [reply Python/stream_reader_sqlrace/atlas_session_writer.py]: The A10 runtime config is used to configure the .NET runtime in pythonnet, refactored to use the A10_INSTALL_PATH now.

File: Python/pylintrc
Comment (owentmfoo) [reply Python/pylintrc:1]: used by pylint for static code analysis.

File: Python/stream_reader_sqlrace/additioanl_requirements.txt
Comment (owentmfoo) [reply Python/stream_reader_sqlrace/additioanl_requirements.txt:1]: fixed

File: Python/stream_reader_sqlrace/main.py (hunk 1 above)
Comment (owentmfoo) [reply Python/stream_reader_sqlrace/main.py]: fixed

File: Python/stream_reader_sqlrace/main.py (hunk 2 above)
Comment (owentmfoo) [reply Python/stream_reader_sqlrace/main.py]: fixed

**Review by salman-mclarenapplied — APPROVED** [review]
